### ISSUE TITLE
Muse Sounds Violin 1 (solo) double stops play staccato when slurred

### ISSUE THREAD

**Issue by Lighwan** [musescore/MuseScore#14039]
https://musescore.com/user/13502736/scores/5582243 var 3

video is self-explanatory
https://user-images.githubusercontent.com/68567124/197863744-d216c9fb-9ed0-4172-8f17-c7792d0b813a.mp4

**Comment by YuenSzeHong** [comment]
you get pag 24 working with muse sounds, I cant get that work for me... crash upon switch

**Comment by Lighwan** [comment]
doesn't happen on newest nightly anyway

**Comment by Lighwan** [comment]
@Tantacrul Going to close this since I'm assuming legato isn't implemented properly yet for violin, so this one's probably redundant.

**Comment by Tantacrul** [comment]
No, this issue should remain open I think. Slurring / phrasing is definitely supported with Muse Sounds.

**Comment by Lighwan** [comment]
Clarified issue
